src/styles/style.css

```diff
@@ -0,0 +1,215 @@
+body {
+  display: flex;
+  height: 100vh;
+  justify-content: center;
+  align-items: center;
+  border: 2px solid red;
+}
+
+.day {
+  box-sizing: border-box;
+  width: 100px;
+  height: 100px;
+  background-color: #eee;
+  border: 1px solid black;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+  color: black;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.month {
+  /* counter-reset: section; */
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  border: 3px solid black;
```

Comment:
The border style here should be consistent with the rest of the design. If a border is necessary, consider defining a class that encapsulates the border styling and apply it where needed.